src/algorithm/structures/mod.rs

```diff
@@ -0,0 +1,6 @@
+mod sensitive;
+
+pub use sensitive_data::SensitiveData;
+pub mod sensitive_data {
+    pub use super::sensitive::data::*;
+}
```

Comment:
I am personally not a big fan of renaming and typedefs 😬 I think it is then harder to map the code structure to the different names. We could also simply do `pub mod sensitive;` here and let users access items inside `data` with:
```
use tss_esapi::algorithm::structures::sensitive::data::...;
```
not so much longer than:
```
use tss_esapi::algorithm::structures::sensitive_data::...;
```
and eventually rename in their code, if they want?

Reply:
As I said before. I did not how to go about this so I checked how it was done in the std. And my goto example is always collections. But I have no real strong opinions regarding this. As long as it is consistent every where I am fine with it. So just tell me how you want it and I will change it.

Reply:
I guess this is fine to keep it like that! I think that they did that in std collections because the last module name is very common and might cause conflicts when imported, like with `data` here for us! So it is probably a good idea to keep `sensitive_data` instead.